contracts/FractalModule.sol

```diff
@@ -0,0 +1,72 @@
+pragma solidity ^0.8.0;
+
+import "@gnosis.pm/zodiac/contracts/core/Module.sol";
+import "@gnosis.pm/safe-contracts/contracts/common/Enum.sol";
+
+contract FractalModule is Module {
```

Comment:
Perhaps create an `IFractalModule` that this inherits from (which would assist in ERC165 `supportsInterface` overriding, which should also be on this contract)?